lib/testLauncher/composeConfig.js

```diff
@@ -109,6 +108,52 @@ function findExtendConfigs(defaultConfigObject, extendPath, filePath, foundPaths
 	return mainConfigFile;
 }
 
+/**
+ * @description Search for configuration files by given path and filename without extension. Can search recursevly.
+ * @param {String} pathToSearch path to directory to search
+ * @param {String} filename base filename without extension
+ * @param {Boolean} shouldSearchDeep boolean value which shows is function should go back to 1 folder up. If true - goes up to the root directory.
+ */
+function deepDirectorySearch(pathToSearch, filename, shouldSearchDeep) {
+	const CONFIG_FORMATS = [
```

Comment:
Pls move this constant out of the function.

Reply:
Wouldn't it create a side effect for this function?